evap/evaluation/models.py

```diff
@@ -443,6 +443,38 @@ class State:
         verbose_name=_("wait for grade upload before publishing"), default=True
     )
 
+    @property
+    def has_exam(self):
+        return self.course.evaluations.filter(name_de="Klausur", name_en="Exam").exists()
+
+    def make_exam_evaluation(
```

Comment:
What do you think of `create_exam_evaluation`? I think that follows django wording (of Manager.create()) more.